en/docs/learn/api-security/oauth2/oauth2-scopes/scope-whitelisting.md

```diff
@@ -2,11 +2,11 @@
 
 A scope is not always used for controlling access to a resource. You can also use it to simply mark an access token. There are scopes that cannot be associated to roles (e.g., openid, device\_). Such scopes do not have to have roles associated with them. Skipping role validation for scopes is called scope whitelisting.
 
-If you do not want a role validation for a scope in an API's request, add the scope as `white_listed_scopes` in the `<APIM_HOME>/repository/conf/deployment.toml` file and restart the server. It will be whitelisted. For example,
+If you do not want a role validation for a scope in an API's request, add the scope as `allowed_scopes` in the `<API-M_HOME>/repository/conf/deployment.toml` file and restart the server. It will be whitelisted. For example,
```

Comment:
Shouldn't the title of the page and even the word "whitelisted" in the sentence "It will be whitelisted." changed if we are moving away from this terminology?

Reply:
Changed "whitelist" to "allowlist" in https://github.com/wso2/docs-apim/pull/1815/commits/c99d7be9433d7804f3c7b6db1efb08cfe49e9cf8